### ISSUE TITLE
a11y update in 6.0.x doesn't render accessibility results

### ISSUE THREAD

**Issue by sleblang** [storybookjs/storybook#12095]
**Describe the bug**
Received the following console error after updating to version 6.0.12:
`withA11y(options) is deprecated, please configure addon-a11y using the addParameter api:`

Clicking on the Accessibility Panel in a story at the bottom/side results in the panel stating 'Initializing' 

**Expected behavior**
Accessibility results should be shown

**Screenshots**
![Screen Shot 2020-08-17 at 2 38 29 PM](https://user-images.githubusercontent.com/2072042/90447135-88eae380-e097-11ea-9503-65f9e7f53904.png)

**Code snippets**
Updated my code in preview.js to accommodate for the new syntax: 
`addParameters({
  a11y: {
    element: '#root',
    manual: false
  }`

**System:**
>   System:
>     OS: macOS Mojave 10.14.6
>     CPU: (12) x64 Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
>   Binaries:
>     Node: 12.16.1 - ~/.nvm/versions/node/v12.16.1/bin/node
>     Yarn: 1.22.4 - ~/.yarn/bin/yarn
>     npm: 6.14.7 - ~/.nvm/versions/node/v12.16.1/bin/npm
>   Browsers:
>     Chrome: 84.0.4147.125
>     Firefox: 76.0.1
>     Safari: 13.1
>   npmPackages:
>     @storybook/addon-a11y: ^6.0.12 => 6.0.12 
>     @storybook/addon-actions: ^6.0.12 => 6.0.12 
>     @storybook/addon-docs: ^6.0.12 => 6.0.12 
>     @storybook/addon-info: ^5.3.19 => 5.3.19 
>     @storybook/addon-knobs: ^6.0.12 => 6.0.12 
>     @storybook/addon-links: ^6.0.12 => 6.0.12 
>     @storybook/addon-storysource: ^6.0.12 => 6.0.12 
>     @storybook/addon-viewport: ^6.0.12 => 6.0.12 
>     @storybook/addons: ^6.0.12 => 6.0.12 
>     @storybook/react: ^6.0.12 => 6.0.12

**Comment by basher** [comment]
Thought it might be worth adding a link to the [deprecated withA11y decorator bug](https://github.com/storybookjs/storybook/issues/11929) that I've already raised.

**Comment by sleblang** [comment]
> Thought it might be worth adding a link to the [deprecated withA11y decorator bug](https://github.com/storybookjs/storybook/issues/11929) that I've already raised.

Thanks. I followed this suggestion  - https://github.com/storybookjs/storybook/issues/11929#issuecomment-672998494
